src/components/standalone/firewall/CreateOrEditPortForwardDrawer.vue

```diff
@@ -0,0 +1,389 @@
+    error.value.notificationTitle = isEditing
+      ? t('error.cannot_edit_port_forward')
+      : t('error.cannot_create_port_forward')
+
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+  } finally {
+    isSubmittingRequest.value = false
+  }
+}
+</script>
+
+<template>
+  <NeSideDrawer
+    :is-shown="isShown"
+    @close="close()"
+    :closeAriaLabel="t('standalone.shell.close_side_drawer')"
+    :title="
+      id
+        ? t('standalone.port_forward.edit_port_forward')
+        : t('standalone.port_forward.add_port_forward')
+    "
+  >
+    <NeInlineNotification
+      v-if="error.notificationTitle"
+      :title="error.notificationTitle"
+      :description="error.notificationDescription"
+      class="mb-6"
+      kind="error"
+    />
+    <NeSkeleton v-if="loading || firewallConfig.loading" :lines="10" />
+    <div v-else class="flex flex-col gap-y-6">
+      <NeToggle :label="t('standalone.port_forward.status')" v-model="enabled" />
+      <NeTextInput
+        :label="t('standalone.port_forward.name')"
+        v-model="name"
+        :invalid-message="validationErrorBag.getFirstFor('name')"
+      />
+      <NeCombobox
+        :label="t('standalone.port_forward.protocols')"
+        :placeholder="t('standalone.port_forward.choose_protocol')"
+        :multiple="true"
+        :options="supportedProtocols"
+        v-model="protocols"
+        :invalid-message="validationErrorBag.getFirstFor('protocols')"
+      />
+      <NeTextInput
+        :label="t('standalone.port_forward.source_port')"
+        v-model="sourcePort"
+        :invalid-message="validationErrorBag.getFirstFor('sourcePort')"
+        type="number"
+      />
+      <NeTextInput
+        :label="t('standalone.port_forward.destination_address')"
+        v-model="destinationIP"
+        :invalid-message="validationErrorBag.getFirstFor('destinationIP')"
+      >
+        <template #tooltip
+          ><NeTooltip
+            ><template #content>{{
+              t('standalone.port_forward.destination_address_tooltip')
+            }}</template></NeTooltip
+          ></template
+        >
+      </NeTextInput>
+      <NeCombobox
+        :label="t('standalone.port_forward.destination_zone')"
+        :placeholder="t('standalone.port_forward.choose_zone')"
+        :options="supportedDestinationZones"
+        :selected-label="t('standalone.port_forward.any_zone')"
+        v-model="destinationZone"
+      />
+      <NeTextInput
+        :label="t('standalone.port_forward.destination_port')"
+        v-model="destinationPort"
+        :invalid-message="validationErrorBag.getFirstFor('destinationPort')"
+        type="number"
+      />
+      <div>
+        <NeButton kind="tertiary" @click="showAdvancedSettings = !showAdvancedSettings">
+          {{ t('standalone.port_forward.advanced_settings') }}
+          <template #suffix>
+            <font-awesome-icon
+              :icon="['fas', showAdvancedSettings ? 'angle-up' : 'angle-down']"
```

Comment:
Advanced settings is already present in other views and is a little bit different. For example it uses chevron icon (we hopefully implement a component for this in the future). Here is an example: https://github.com/NethServer/nethsecurity-ui/blob/main/src/components/standalone/interfaces_and_devices/ConfigureDeviceDrawer.vue#L1298-L1312